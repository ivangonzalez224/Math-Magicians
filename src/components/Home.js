const Home = () => (
  <div className="main-home">
    <h2>Welcome to our page !</h2>
    <p>This calculator app provides simple mathematical functions in a beautifully designed app.</p>
    <p>
      <span>• Perform basic calculations such as addition, subtraction,</span>
      <span> multiplication, division and porcentage.</span>
    </p>
    <p>
      <span>• Find a wide range of inspirational succes quotes if you</span>
      <span> need an extra push to get stuff done.</span>
    </p>
  </div>
);

export default Home;
