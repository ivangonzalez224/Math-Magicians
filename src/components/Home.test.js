import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from './Home';

describe('Home component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Home />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
