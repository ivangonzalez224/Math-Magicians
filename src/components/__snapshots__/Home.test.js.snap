// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div
  className="main-home"
>
  <h2>
    Welcome to our page !
  </h2>
  <p>
    This calculator app provides simple mathematical functions in a beautifully designed app.
  </p>
  <p>
    <span>
      • Perform basic calculations such as addition, subtraction,
    </span>
    <span>
       multiplication, division and porcentage.
    </span>
  </p>
  <p>
    <span>
      • Find a wide range of inspirational succes quotes if you
    </span>
    <span>
       need an extra push to get stuff done.
    </span>
  </p>
</div>
`;
